VARIABLES.md

```diff
@@ -540,6 +556,14 @@ Default:  "{{ ssl_mutual_auth_enabled }}"
 
 ***
 
+### schema_registry_log_dir
+
+Only use to customize directory Schema Registry writes log files to. Default location is /var/log/confluent/schema-registry. NOTE- schema_registry.appender_log_path is deprecated
```

Comment:
Modify this variable to customize the directory that the Schema Registry writes log files to. Default location is /var/log/confluent/schema-registry. NOTE- schema_registry.appender_log_path is deprecated